<!DOCTYPE html>
<html lang="en">
<head>

<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<meta name="description" content="Personal Website">
<meta name="keywords" content="Epigenomics, Epigenetics, Michael, Vinyard, Luca, Pinello, Gad, Getz, Broad, Institute, science, chemical biology, LSD1, single cell, virtual reality, machine learning, artificial intelligence" />
<meta name="Michael Edward Vinyard" content="Personal Website">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

<title>Michael E. Vinyard</title>

<!--

Template 2085 Neuron

http://www.tooplate.com/view/2085-neuron

-->
<link rel="stylesheet" href="css/bootstrap.min.css">
<!-- <link rel="stylesheet" href="css/font-awesome.min.css"> -->
<link href="fontawesome/css/all.css" rel="stylesheet"> <!--load all styles -->
<!-- Main css -->
<link rel="stylesheet" href="css/style.css">
<link href="https://fonts.googleapis.com/css?family=Lora|Merriweather:300,400" rel="stylesheet">

</head>
<body>

<!-- PRE LOADER -->

<div class="preloader">
     <div class="sk-spinner sk-spinner-wordpress">
          <span class="sk-inner-circle"></span>
     </div>
</div>

<!-- Navigation section  -->

<div class="navbar navbar-default navbar-static-top" role="navigation">
     <div class="container">

          <div class="navbar-header">
               <button class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                    <span class="icon icon-bar"></span>
                    <span class="icon icon-bar"></span>
                    <span class="icon icon-bar"></span>
               </button>
               <a href="index.html" class="navbar-brand">Vinyard, Michael E.</a>
          </div>
          <div class="collapse navbar-collapse">
               <ul class="nav navbar-nav navbar-right">
                    <!-- <li class="active"><a href="#home" class="smoothScroll">Home</a></li> -->
                    <li><a href="#about" class="smoothScroll"><strong>About</strong></a></li>
                    <li><a href="#pubs" class="smoothScroll"><strong>Publications & Software</strong></a></li>
                    <li><a href="#blog" class="smoothScroll"><strong>Blog</strong></a></li>
                    <li><a href="#uniquecontact" class="smoothScroll"><strong>Contact</strong></a></li>
               </ul>
          </div>
  </div>
</div>

<!-- Home Section -->

<section id="home" class="main-home parallax-section">
     <div class="overlay_home"></div>
     <div id="particles-js"></div>
     <div class="container">
          <div class="row">
               <div class="col-md-12 col-sm-12">
                    <div class = "titlehead"><h1 class = 'titlehead'></h1></div>
                    <h4>My personal website.</h4>
                    <a href="#blog" class="smoothScroll btn btn-default">Discover More</a>
               </div>

          </div>
     </div>
</section>


<!-- About Home Section -->

<section id="home" class="main-about parallax-section">
     <div class="overlay"></div>
     <div class="container">
          <div class="row">
            <div class = "subtitle"><h1 class = 'subtitle'></h1></div>
        </div>
     </div>
</section>

<!-- About Section -->

<section id="about">
     <div class="container">
          <div class="row">

               <div class="col-md-offset-1 col-md-10 col-sm-12">

                    <div class="col-md-4 col-sm-4">
                         <h2>sc-multi-omics</h2>
                         <p>While methods to interrogate all levels of the central dogma in a single cell currently elude our capabilities, we are capable of analyzing one mode of DNA-derived analysis in combination with single-cell transcriptomic analysis. I am particularly interested in exploring this area in primary patient tissues.</p>
                    </div>

                    <div class="col-md-4 col-sm-4">
                         <h2>Disease</h2>
                         <p>While methods to interrogate all levels of the central dogma in a single cell currently elude our capabilities, we are capable of analyzing one mode of DNA-derived analysis in combination with single-cell transcriptomic analysis. I am particularly interested in exploring this area in primary tissues from cancer patients.</p>
                    </div>

                    <div class="col-md-4 col-sm-4">
                         <h2>Extracurriculars</h2>
                         <p>I like distance running on occasion and I am currently trying to incorporate more machine learning into my research projects.</p>

                        <p>I am also pursuing several side projects. I am really interested in dynamic data visualizations, especially those that seek to make intuitive data that matters widely to the public. In addition, I have also been accumulating components for DIY Smart-Home gadgets powered by Raspberry Pi.</p>
                    </div>

                    <div class="col-md-6 col-sm-6">
                        <h3>A brief introduction</h3>
                        <p>Welcome to my personal website and blog. I like to write about current science, philosophy, and running. I enjoy thinking hard about problems in science; writing is the only way to make sense of your thoughts and get feedback. In this way, we might advance our understanding of the problems we wish to solve.</p>

              					<div class="blog-post-des"><a href = "cv_mev_2019_JUL_01.pdf" target="_blank" class="btn btn-default">Download my CV Here</a></div>

                    </div>

                    <div class="col-md-6 col-sm-6">
                         <img src="images/mev_color.jpeg" class="img-responsive" alt="About Image">
                    </div>

                    <div class="clearfix"></div>

                    <div class="col-md-4 col-sm-4">
                         <img src="images/dogma.jpeg" class="img-responsive" alt="Blog Image">
                         <h3>Chromatin</h3>
                         <p>As we are coming to understand in recent decades, chromatin signalling likely acts as a platform for many biological phenomena. In addition to the chromatin-signalling defects that can spur disease, I am also interested in the basic functions of life from this angle. </p>
                    </div>

                    <div class="col-md-4 col-sm-4">
                         <img src="images/AML.jpg" class="img-responsive" alt="Blog Image">
                         <h3>Disease Mechanism</h3>
                         <p>I am investigating cancer-driving mechanism in single cells. So far in my PhD, I have gained experience in analyzing hematopoietic cancers.</p>
                    </div>

                    <div class="col-md-4 col-sm-4">
                         <img src="images/landscape_head.jpeg" class="img-responsive" alt="Blog Image">
                         <h3>Data Science</h3>
                         <p>I am a machine learning novice (to the extreme), however I am excited to learn as much as I can to implement novel ideas in biological research.</p>
                    </div>
               </div>

          </div>
     </div>
</section>

<!-- Publications Section -->

<section id="home" class="main-about parallax-section">
     <div class="overlay"></div>
     <div class="container">
       <div class="row">
         <div class = "subtitle2"><h1 class = 'subtitle2'></h1></div>
     </div>
     </div>
</section>


<!-- Blog Section -->

<section id="blog">
     <div class="container">
          <div class="row">
               <div class="col-md-offset-1 col-md-10 col-sm-12">
                    <div class="blog-post-thumb">
                         <div class="blog-post-image row">
<!-- top of pub -->
                          <div class="blog-post-title">
                               <h3><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1854-5">2. Assessment of computational methods for the analysis of single-cell ATAC-seq data</a></h3>
                          </div>

                          <!-- <div class="col-md-6 col-sm-6">
                               <div class="gallery-thumb">
                                    <a href="https://github.com/pinellolab/scATAC-benchmarking" target="_blank">
                                         <img src="images/scATAC.png" class="img-responsive" alt="Gallery Image">
                                    </a>
                               </div>
                          </div> -->

                         </div>

                         <div class="blog-post-des">

                           <div class="col-md-2 col-sm-6">

                           <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1854-5" target="_blank">
                                <img width = 75% src="images/gb.jpg" class="img-responsive" alt="Gallery Image">
                           </a></div>
                           <!-- <div class="col-md-8 col-sm-6">
                           <a href="https://github.com/pinellolab/scATAC-benchmarking" target="_blank">
                                <img src="images/scATAC.png" class="img-responsive" alt="Gallery Image">
                           </a>

                         </div> -->

                              <p>Huidong Chen, Caleb Lareau<sup>†</sup>, Tommaso Andreani<sup>†</sup>, <strong>Michael E. Vinyard<sup>†</sup></strong>, Sara P. Garcia, Kendell Clement, Miguel A. Andrade-Navarro, Jason D. Buenrostro & Luca Pinello. <strong>Assessment of computational methods for the analysis of single-cell ATAC-seq data</strong>. <i>Genome Biol.</i> 20, 241 (2019).</p>
                              <br></br>
                              <!-- <h3><strong>Abstract</strong>:</h3>
                              <p><strong>Background</strong>: Recent innovations in single-cell Assay for Transposase Accessible Chromatin using sequencing
                                (scATAC-seq) enable profiling of the epigenetic landscape of thousands of individual cells. scATAC-seq data analysis
                                presents unique methodological challenges. scATAC-seq experiments sample DNA, which, due to low copy
                                numbers (diploid in humans), lead to inherent data sparsity (1–10% of peaks detected per cell) compared to
                                transcriptomic (scRNA-seq) data (10–45% of expressed genes detected per cell). Such challenges in data generation
                                emphasize the need for informative features to assess cell heterogeneity at the chromatin level.</p>
                                <p><strong>Results</strong>: We present a benchmarking framework that is applied to 10 computational methods for scATAC-seq on
                                13 synthetic and real datasets from different assays, profiling cell types from diverse tissues and organisms. Methods
                                for processing and featurizing scATAC-seq data were compared by their ability to discriminate cell types when
                                combined with common unsupervised clustering approaches. We rank evaluated methods and discuss
                                computational challenges associated with scATAC-seq analysis including inherently sparse data, determination of
                                features, peak calling, the effects of sequencing coverage and noise, and clustering performance. Running times
                                and memory requirements are also discussed.</p> -->

                                <a href=href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1854-5" class="btn btn-default">Read it Here</a>
                                <a href=href="https://github.com/pinellolab/scATAC-benchmarking" class="btn btn-default">GitHub Repository</a>
                                <br></br>
                                <p><strong>Conclusions</strong>: This reference summary of scATAC-seq methods offers recommendations for best practices with
                                consideration for both the non-expert user and the methods developer. Despite variation across methods and
                                datasets, SnapATAC, Cusanovich2018, and cisTopic outperform other methods in separating cell populations of
                                different coverages and noise levels in both synthetic and real datasets. Notably, SnapATAC is the only method able
                                to analyze a large dataset (> 80,000 cells).</p>
                                <img src="images/Figure2.png" class="img-responsive" alt="Blog Image">

                         </div>

                    </div>

<!-- Bottom of pub -->

<!-- top of pub -->
                          <div class="blog-post-title">
                               <h3><a href="https://www.nature.com/articles/s41589-019-0263-0">1. CRISPR-suppressor scanning reveals a nonenzymatic role of LSD1 in AML</a></h3>

                           <div class="col-md-2 col-sm-6">

                           <a href="https://www.nature.com/articles/s41589-019-0263-0" target="_blank">
                                <img width = 75% src="images/may.png" class="img-responsive" alt="Gallery Image">
                           </a></div>

                         <div class="blog-post-des">
                              <p><strong>2</strong>. Michael E. Vinyard, Cindy Su, Allison P. Siegenfeld, Amanda L. Waterbury, Allyson M. Freedy, Pallavi M. Gosavi, Yongho Park, Eugene E. Kwan, Benjamin D. Senzer, John G. Doench, Daniel E. Bauer, Luca Pinello & Brian B. Liau. <strong>CRISPR-suppressor scanning reveals a nonenzymatic role of LSD1 in AML</strong>. <i>Genome Biol.</i> 20, 241 (2019).</p>
                              <br>
                              <a href=href="https://www.nature.com/articles/s41589-019-0263-0" class="btn btn-default">Read it Here</a>
                              <img src="images/ncb_abstract.jpeg" class="img-responsive" alt="Blog Image">
                              <br></br>
                              <p><strong>Abstract</strong>: Understanding the mechanism of small molecules is a critical challenge in chemical biology and drug discovery. Medicinal chemistry is essential for elucidating drug mechanism, enabling variation of small molecule structure to gain structure–activity relationships (SARs). However, the development of complementary approaches that systematically vary target protein structure could provide equally informative SARs for investigating drug mechanism and protein function. Here we explore the ability of CRISPR–Cas9 mutagenesis to profile the interactions between lysine-specific histone demethylase 1 (LSD1) and chemical inhibitors in the context of acute myeloid leukemia (AML). Through this approach, termed CRISPR-suppressor scanning, we elucidate drug mechanism of action by showing that LSD1 enzyme activity is not required for AML survival and that LSD1 inhibitors instead function by disrupting interactions between LSD1 and the transcription factor GFI1B on chromatin. Our studies clarify how LSD1 inhibitors mechanistically operate in AML and demonstrate how CRISPR-suppressor scanning can uncover novel aspects of target biology.</p>
                         </div>

                      </div>

                    </div>
                  </div>
               </div>
<!-- Bottom of pub -->

          </div>
     </div>
</section>




<!-- About Home Section -->

<section id="home" class="main-about parallax-section">
     <div class="overlay"></div>
     <div class="container">
          <div class="row">

               <div class="col-md-12 col-sm-12">
                    <div id = "uniqueblog" class = "subtitle3"><h1 class = 'subtitle3'></h1></div>
               </div>

          </div>
     </div>



</section>
<br></br><br></br>



<!-- About Section -->

<section id="uniqueblog">
     <div class="container">
          <div class="row">

               <div class="col-xs-12 col-sm-8 col-md-6 col-lg-8">

                 <div id="retainable-rss-embed"
                 data-rss="https://medium.com/feed/@vinyard_m"
                 data-maxcols="2"
                 data-layout="slider"
                 data-poststyle="inline"
                 data-readmore="Read the rest"
                 data-buttonclass="btn btn-primary"
                 data-offset="-100"></div>

               </div>

                 <!-- Twitter Section, part of the latest blog post-->
                 <div class="col-xs-12 col-sm-8 col-md-6 col-lg-4" data-animate-effect="fadeInLeft">
                                                    <div ALIGN = RIGHT class = "offsetcol-md-8 col-sm-12"><a class="twitter-timeline" data-width="600" data-height="600" data-theme="light"
                                                      href="https://twitter.com/vinyard_m?ref_src=twsrc%5Etfw">Tweets by vinyard_m</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
                                                    </div>
                                                  </div>
                 <!-- Twitter Section, part of the latest blog post-->

          </div>
     </div>
</section>



<!-- Footer Section -->

<footer>
     <div id="uniquecontact" class="container">
          <div class="row">

               <div class="col-md-5 col-md-offset-1 col-sm-6">
                    <h3>Contact Me</h3>
                    <p>I hope you enjoy my website! Let me know if you have suggestions. You can find my various social medias and laboratory websites below.</p>

                    <p><i><strong>Important</strong>: All views expressed on this website are my own and not those of any labs or institutions I am affiliated with.</i></p>
                    <div class="footer-copyright">
                         <p>Copyright &copy; 2020 Michael E. Vinyard</p>
                    </div>
               </div>

               <div class="col-md-4 col-md-offset-1 col-sm-6">
                    <br></br>
                    <br>
                    <p><i class="fa fa-globe"></i>70 Ames Street, Cambridge, MA. 02142</p>
                    <p><i class="fa fa-phone"></i> 617-714-8107</p>
                    <p><i class="fa fa-save"></i> mvinyard@broadinstitute.org</p>
               </div>

               <div class="clearfix col-md-12 col-sm-12">
                    <hr>
               </div>

               <div class="col-md-12 col-sm-12 col-lg-12">
                    <ul class="social-icon">
              					<li><a href="http://pinellolab.org" class="fas fa-code"></a></li>
              					<li><a href="https://www.broadinstitute.org/labs/getz" class="fa fa-tv"></a></li>
              					<li><a href="http://www.liaulab.org" class="fa fa-flask"></a></li>
              					<li><a href="https://github.com/mvinyard?tab=repositories"><i class="fab fa-github fa-2x"></i></a></li>
              					<li><a href="https://twitter.com/vinyard_m"><i class="fab fa-twitter fa-2x"></i></a></li>
              					<li><a href="https://www.linkedin.com/in/michaelvinyard/"><i class="fab fa-linkedin fa-2x"></i></a></li>
              					<li><a href="https://www.strava.com/athletes/30499516" class="fab fa-strava fa-3x"></a></li>
                    </ul>
               </div>

          </div>
     </div>
</footer>

<!-- Back top -->
<a href="#back-top" class="go-top"><i class="fa fa-angle-up"></i></a>

<!-- SCRIPTS -->
<script src="js/jquery.js"></script>
<script src="js/bootstrap.min.js"></script>
<script src="js/particles.min.js"></script>
<script src="js/app.js"></script>
<script src="js/jquery.parallax.js"></script>
<script src="js/smoothscroll.js"></script>
<script src="js/custom.js"></script>
<script src="js/typewriter.js"></script>

<script>
window.onscroll = function() {myFunction()};

var navbar = document.getElementById("navbar");
var sticky = navbar.offsetTop;

function myFunction() {
  if (window.pageYOffset >= sticky) {
    navbar.classList.add("sticky")
  } else {
    navbar.classList.remove("sticky");
  }
}</script>

<script src="https://www.retainable.io/assets/retainable/rss-embed/retainable-rss-embed.js"></script>


</body>
</html>
